framework/resource/logresource/resource.go

```diff
@@ -61,58 +62,62 @@ func New(config Config) (*Resource, error) {
 }
 
 func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interface{}, error) {
-	r.logger.LogWithCtx(ctx, "action", "start", "component", "operatorkit", "function", "GetCurrentState")
```

Comment:
I don't mind removing the `action` here, since it was never useful for me. We see when a scope starts or ends by seeing logs about it or not.